torchrl/envs/transforms/transforms.py

```diff
@@ -4826,25 +4826,23 @@ class Hash(UnaryTransform):
         in_keys (sequence of NestedKey): the keys of the values to hash.
         out_keys (sequence of NestedKey): the keys of the resulting hashes.
         in_keys_inv (sequence of NestedKey, optional): the keys of the values to hash during inv call.
-
-            .. note:: If an inverse map is required, a repertoire ``Dict[Tuple[int], Any]`` of hash to value should be
-                passed alongside the list of keys to let the ``Hash`` transform know how to recover a value from a
-                given hash. This repertoire isn't copied, so it can be modified in the same workspace after the
-                transform instantiation and these modifications will be reflected in the map. Missing hashes will be
-                mapped to ``None``.
-
         out_keys_inv (sequence of NestedKey, optional): the keys of the resulting hashes during inv call.
 
     Keyword Args:
         hash_fn (Callable, optional): the hash function to use. If ``seed`` is given,
-            the hash function must accept it as its second argument. Default is
-            ``Hash.reproducible_hash``.
+            the hash function must accept it as its second argument. The output of
+            the hash function must be a tensor. Default is ``Hash.reproducible_hash``.
```

Comment:
I thought it would be difficult to support all output types for `hash_fn`, since it's not obvious how to convert any arbitrary type into a dictionary key. But we could try to remove this restriction if needed